src/lib/bootloader/autoyast_converter.rb

```diff
@@ -207,6 +220,17 @@ def bootloader_from_data(data)
         end
       end
 
+      # only for grub2, not for others
+      GRUB2EFI_BOOLEAN_MAPPING = {
```

Comment:
NP: Should it be a private constant?

Reply:
good point, also others are not private, so lets mark all of them as private.